### PR TITLE
Rename workflow options in metadata

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: database/migration/src/main/scala/cromwell/database/migration/workflowoptions/RenameWorkflowOptionsInMetadata.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import java.sql.{PreparedStatement, ResultSet}
+
+import cromwell.database.migration.custom.BatchedTaskChange
+import cromwell.database.migration.workflowoptions.WorkflowOptionsRenaming._
+import spray.json.{JsObject, _}
+
+/**
+  * Clear the values from encrypted keys in METADATA_ENTRY.
+  */
```
Comment (Horneth) [inline database/migration/src/main/scala/cromwell/database/migration/workflowoptions/RenameWorkflowOptionsInMetadata.scala]: Wrong comment

**Comment by mcovarr** [comment]
:+1:

**Comment by ruchim** [comment]
👍

**Comment by mcovarr** [comment]
with parallel PRs on the other branches too obvs

**Pull request merged**